Applications/ApplicationsLib/ProjectData.h

```diff
@@ -63,7 +64,8 @@ class ProjectData final
 	/// The additional  path is used to find files referenced in the
 	/// configuration.
 	ProjectData(BaseLib::ConfigTree const& config_tree,
-	            std::string const& path);
+	            std::string const& path,
+	            std::string const& output_directory);
```

Comment:
- Maybe `path` should be renamed to `input_path` or `input_directory` to reflect better that this variable is used for input in contrast to the `output_directory`? 
- I would prefer `output_path` instead of `output_directory`.
- For consistency I would either document all parameters or none. At the moment `path` is documented, `output_directory` is not.